src/components/tooltip/TooltipDirective.js

```diff
@@ -294,6 +299,30 @@ goog.require('ga_topic_service');
                 onCloseCB = data.onCloseCB;
               }, 0);
             });
+            // TODO handle vector layer toolip
+            $rootScope.$on('$translateChangeEnd', function() {
+              var tmp, feature, layerBodId, featureId;
+              if (scope.isActive && htmls.length) {
+                cancelRequests();
+                for (var i = 0; i < htmls.length; i++) {
+                  tmp = htmls[i];
+                  feature = tmp.feature;
+                  if (feature) {
+                    layerBodId = feature.layerBodId;
+                    if (layerBodId) {
+                      featureId = feature.id;
+                      (function(i, tmp, layerBodId, featureId) {
+                        getFeaturePopupHtml(layerBodId,
+                            featureId).then(function(response) {
+                          tmp.snippet = $sce.trustAsHtml(response.data);
+                          htmls.splice(i, 1, tmp);
+                        });
+                      })(i, tmp, layerBodId, featureId);
```

Comment:
ok, as a side note, the silce is needed as you want to replace the html snippet.